src/js/components/Video/propTypes.js

```diff
@@ -36,6 +36,7 @@ if (process.env.NODE_ENV !== 'production') {
       volumeDown: PropTypes.string,
       volumeUp: PropTypes.string,
     }),
+    skipInterval: PropTypes.number,
```

Comment:
While we are in here, can we make sure that the propTypes and index.d.ts definitions are at parity and easy to compare?
- first level of properties, can we alphabetize?
- messages properties, can we alphabetize?
- `controls.items` is missing enum prop type

I think that's all.